src/main/java/com/launchdarkly/client/LDClient.java

```diff
@@ -294,47 +294,36 @@ public JsonElement jsonVariation(String featureKey, LDUser user, JsonElement def
     return value;
   }
 
-
   private JsonElement evaluate(String featureKey, LDUser user, JsonElement defaultValue) {
     if (!initialized()) {
       return defaultValue;
     }
     try {
       FeatureFlag featureFlag = config.featureStore.get(featureKey);
-      if (featureFlag != null) {
-        if (config.stream && config.debugStreaming) {
-          FeatureFlag pollingResult = requestor.makeRequest(featureKey, true);
-          if (!featureFlag.equals(pollingResult)) {
-            logger.warn("Mismatch between streaming and polling feature! Streaming: {} Polling: {}", featureFlag, pollingResult);
-          }
-        }
-      } else {
+      if (featureFlag == null) {
```

Comment:
I removed the `debugStreaming` option; I believe it has outlived its usefulness.